wandb/sdk/wandb_artifacts.py

```diff
@@ -466,13 +467,32 @@ def verify(self, root: Optional[str] = None):
             "Cannot call verify on an artifact before it has been logged or in offline mode"
         )
 
-    def save(self) -> None:
+    def save(self, **init_kwargs) -> None:  # type: ignore
```

Comment:
If we do this, we should add details about it to the doc string.  It's rather awkward...  We may want to just skip this for now and only let the user override our defaults with environment variables.

Reply:
Actually, we should make this be an Optional Settings object and it's far less awkward.

Reply:
ok, will change to settings.